README.md

```diff
@@ -53,11 +53,11 @@ Other contracts which makes use of the `EVMRuntime` are:
 It also exists a corresponding runtime implementation on the JS side.
 [You can take a look at the on-chain Verifier unit test on how it's used.](https://github.com/leapdao/solEVM-enforcer/blob/master/test/verifier.js)
 
-[The OffchainStepper mimics the EthereumRuntime contract](https://github.com/leapdao/solEVM-enforcer/blob/master/utils/OffchainStepper.js)
+[The off-chain EVMRuntime mimics the on-chain counterpart](https://github.com/leapdao/solEVM-enforcer/blob/master/utils/EVMRuntime.js)
```

Comment:
we have been using the words `off-chain interpreter` and `on-chain stepper`. maybe we can make it consistent in the doc?

Reply:
🤔 I thought talking about on-chain Runtime and off-chain Runtime is more clear, instead of naming these differently. They are essentially the same

Reply:
well, if they are different, it is harder to confuse them, that is the whole point :)

Reply:
Better? 😄